src/omnibase_core/utils/workflow_executor.py

```
@@ -0,0 +1,545 @@
+            )
+
+            # Emit action for this step
+            action = _create_action_for_step(step, workflow_id)
+            all_actions.append(action)
+
+            # Mark step as completed
+            context.completed_at = datetime.now()
+            completed_steps.append(str(step.step_id))
+            completed_step_ids.add(step.step_id)
+
+        except Exception:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)